sql_generators/experiment_monitoring/templates/experiment_enrollment_aggregates_v1/query.sql

```diff
@@ -25,6 +25,21 @@ WITH
         event_category = 'normandy' AND
         submission_date = @submission_date
     ),
+  {% elif "_cirrus" in app_dataset %}
+  {{ app_dataset }} AS (
+    SELECT
+      submission_timestamp AS `timestamp`,
+      event.category AS `type`,
+      mozfun.map.get_key(event.extra, 'experiment') AS experiment,
```

Comment:
@scholtzan I just wanted to confirm that we are OK using mozfun UDFs here. I see elsewhere that we use `CAST(event.extra[safe_offset(i)].value AS STRING) AS branch,` so I wanted to make sure we don't need to change to that for all of these templates.

Reply:
Ah yes, good catch! UDFs are not supported in materialized views: https://cloud.google.com/bigquery/docs/materialized-views-create#unsupported_sql_features